overlord/snapstate/snapmgr_test.go

```diff
@@ -140,6 +140,15 @@ func verifyInstallUpdateTasks(c *C, curActive bool, ts *state.TaskSet, st *state
 	return n
 }
 
+func (s *snapmgrTestSuite) TestLastIndexFindsLast(c *C) {
+	snapst := &snapstate.SnapState{Sequence: []*snap.SideInfo{
+		{Revision: snap.R(7)},
+		{Revision: snap.R(11)},
```

Comment:
The question is probably about the fact we have multiple 11 entries here. I also assumed earlier that this state would only be reached by mistake.